test-integration/operator-tests/src/main/java/org/infinispan/Infinispan.java

```diff
@@ -68,7 +68,9 @@ public void waitFor() {
 
          List<Condition> conditions = infinispanObject.getStatus().getConditions();
          if (conditions != null) {
-            Condition wellFormed = conditions.stream().filter(c -> "wellFormed".equals(c.getType())).findFirst().orElse(null);
+            Condition wellFormed = conditions.stream()
+                    .filter(c -> "WellFormed".equals(c.getType()) || "wellFormed".equals(c.getType()))
```

Comment:
@Crumby it would be nice to use equalsIgnoreCase for this to don't care about in the fiuture

Reply:
@dmvolod Actually I'm thinking of hardcoding only the `WellFormed` version. There are conventions that apply for the status and this is itself is way to discover a regression.

My original idea was to make the tests backward compatible but they contain new tests not valid for older releases and older version of the tests is available under corresponding repository tag. What do you think?

Reply:
Good, @Crumby agreed